packages/fx-core/src/component/generator/generatorAction.ts

```diff
@@ -177,39 +73,69 @@ export const unzipAction: GeneratorAction = {
   },
 };
 
-export const fetchTemplateFromLocalAction: GeneratorAction = {
-  name: GeneratorActionName.FetchTemplateZipFromLocal,
+export const LocalTemplateAction: GeneratorAction = {
+  name: GeneratorActionName.LocalTemplate,
   run: async (context: GeneratorContext) => {
     if (context.outputs?.length) {
```

Comment:
In the new design, why do we still need output to pass zip content?

Reply:
The logic here is so that when the RemoteTemplateAction downloads successfully, there is no need to execute the LocalTemplateAction.